e2e-tests/cross-site-sharded/run

```diff
@@ -0,0 +1,175 @@
+			  {"affinity":{"antiAffinityTopologyKey": "none"},"arbiter":{"affinity":{"antiAffinityTopologyKey": "none"},"enabled":false,"size":1},"expose":{"enabled":true,"exposeType":"LoadBalancer"},"externalNodes":[{"host":"'${replica_rs1_0_endpoint}'"},{"host":"'${replica_rs1_1_endpoint}'","port":27017,"priority":1,"votes":1},{"host":"'${replica_rs1_2_endpoint}'", "port":27017,"priority":1,"votes":1}],"name":"rs1","nonvoting":{"affinity":{"antiAffinityTopologyKey":"none"},"enabled":false,"podDisruptionBudget":{"maxUnavailable":1},"resources":{"limits":{"cpu":"300m","memory":"0.5G"},"requests":{"cpu":"300m","memory":"0.5G"}},"size":3,"volumeSpec":{"persistentVolumeClaim":{"resources":{"requests":{"storage":"1Gi"}}}}},"podDisruptionBudget":{"maxUnavailable":1},"resources":{"limits":{"cpu":"300m","memory":"0.5G"},"requests":{"cpu":"300m","memory":"0.5G"}},"size":3,"volumeSpec":{"persistentVolumeClaim":{"resources":{"requests":{"storage":"3Gi"}}}}}
+			  ],
+			  "sharding":{"configsvrReplSet":{ "externalNodes": [{"host":"'${replica_cfg_0_endpoint}'","priority":1,"votes":1 },{"host":"'${replica_cfg_1_endpoint}'", "priority":1,"votes":1},{"host":"'${replica_cfg_2_endpoint}'"}]}}
+			  }
+		}'
+
+sleep 60
+kubectl_bin config set-context $(kubectl_bin config current-context) --namespace="$replica_namespace"
+desc 'check if all 3 Pods started'
+wait_for_running $replica_cluster-rs0 3
+wait_for_running $replica_cluster-cfg 3 "false"
+
+desc 'Write data, read from all'
+
+run_mongos \
+	'db.createUser({user:"myApp",pwd:"myPass",roles:[{db:"myApp",role:"readWrite"}]})' \
+	"userAdmin:userAdmin123456@$main_cluster-mongos.$namespace"
+sleep 2
+run_mongos \
+	'use myApp\n db.test.insert({ x: 100500 })' \
+	"myApp:myPass@$main_cluster-mongos.$namespace"
+
+sleep 10 # for minikube
+desc "Compare data"
+compare_mongos_cmd "find" "myApp:myPass@$main_cluster-mongos.$namespace"
+
+desc 'Test failover'
+kubectl_bin config set-context $(kubectl_bin config current-context) --namespace="$namespace"
+kubectl_bin delete psmdb $main_cluster
+sleep 120
+
+run_script_mongos "${test_dir}/disaster_recovery.js" "clusterAdmin:clusterAdmin123456@$replica_rs0_0_endpoint" "mongodb" ":27017"
+run_script_mongos "${test_dir}/disaster_recovery.js" "clusterAdmin:clusterAdmin123456@$replica_rs1_0_endpoint" "mongodb" ":27017"
+run_script_mongos "${test_dir}/disaster_recovery.js" "clusterAdmin:clusterAdmin123456@$replica_cfg_0_endpoint" "mongodb" ":27017"
```

Comment:
We should probably also switch to the `unmanaged: false`. Do we need to apply disaster_recovery.js to all 3 cfg or just one would maybe be ok?

Reply:
I added unmanaged: false settings. I didn't catch why "3 cfg". We need to do it for each RS.